meorg_client/cli.py

```diff
@@ -109,19 +109,31 @@ def list_endpoints():
 
 @click.command("upload")
 @click.argument("file_path", nargs=-1)
-def file_upload(file_path: tuple):
+@click.option(
+    "--attach_to",
+    default=None,
+    help="Supply a model output id to immediately attach the file to.",
+)
+def file_upload(file_path, attach_to=None):
     """
     Upload a file to the server.
 
     Prints Job ID on success, which is used by file-status to check transfer status.
+
+    If attach_to is used then no ID is returned.
     """
     client = _get_client()
 
     # Upload the file, get the job ID
-    response = _call(client.upload_files, files=list(file_path))
-    files = response.get("data").get("files")
-    for f in files:
-        click.echo(f.get("file"))
+    response = _call(client.upload_files, files=list(file_path), attach_to=attach_to)
+
+    # Different logic if we are attaching to a model output immediately
+    if not attach_to:
+        files = response.get("data").get("files")
+        for f in files:
+            click.echo(f.get("file"))
```

Comment:
Just curious, why not echo the list of files in the `attach_to == True` case?

Reply:
Thanks @SeanBryan51, the reasoning is that outputting the file IDs does not indicate that the attach command actually worked, only that it successfully uploaded.

Regardless, it occurs to me that the call will simply fail with a non-zero exit status if the attachment fails, so I see no harm in adding the output of file ids.

I have just pushed a new commit for this, also adding the attachment code to the parallel version of the upload.

Tests passing, closing issue.